src/core/event.cpp

```diff
@@ -267,9 +267,9 @@ QJsonObject Event::buildObjectEventJSON()
     return eventObj;
 }
 
-QJsonObject Event::buildWarpEventJSON(QMap<QString, QString> *mapNamesToMapConstants)
+OrderedJson::object Event::buildWarpEventJSON(QMap<QString, QString> *mapNamesToMapConstants)
 {
-    QJsonObject warpObj;
+    OrderedJson::object warpObj;
     warpObj["x"] = this->getU16("x");
```

Comment:
Does `OrderedJson` behave the same way as Qt's library when accessing an element that is non-existent?  (i.e. it doesn't crash, and returns the default value)

Reply:
Yes